.gitignore

```diff
@@ -34,3 +34,7 @@
 yarn-debug.log*
 .yarn-integrity
 
+# DS_Store
```

Comment:
traditionally DS_Store goes in your global git ignore since is something we want to leave out for every project.

Ways to add things to your global git ignore:
https://help.github.com/articles/ignoring-files/

Reply:
look at sample global gitignore files as well for other fun things to add that can be specific to editors

Reply:
gotcha, will remove from local gitignore